server/post.go

```diff
@@ -64,15 +64,18 @@ func (p *Plugin) buildAnalyticMsg() (string, *model.AppError) {
 		parametersURLPie.Add(c.displayName, fmt.Sprintf("%d", c.nb))
 	}
 	urlPie.RawQuery = parametersURLPie.Encode()
-	pie := "![](" + urlPie.String() + ") "
+	pie := fmt.Sprintf("![channels pie chart](%s) ", urlPie.String())
 
 	urlBar, _ := url.Parse("http://127.0.0.1:8065/plugins/com.github.manland.mattermost-plugin-analytics/bar.svg")
 	parametersURLBar := url.Values{}
-	for _, c := range data.users {
+	for index, c := range data.users {
+		if index > 10 {
```

Comment:
why?

Reply:
it's the number max to display a good bar chart. If we have 200 users, the chart will be unreadable. I have added a constant to be more comprehensive.